src/hypothesis/searchstrategy/numbers.py

```diff
@@ -90,14 +90,17 @@ def do_draw(self, data):
 
 
 NASTY_FLOATS = [
-    0.0, 0.5, 1.0 / 3, 10e6, 10e-6, 1.175494351e-38, 2.2250738585072014e-308,
+    0.0, 0.5, 1.1, 1.5, 1.9, 1.0 / 3, 10e6, 10e-6, 1.175494351e-38,
+    2.2250738585072014e-308,
     1.7976931348623157e+308, 3.402823466e+38, 9007199254740992, 1 - 10e-6,
     2 + 10e-6, 1.192092896e-07, 2.2204460492503131e-016,
 
 ] + [float('inf'), float('nan')] * 5
 NASTY_FLOATS.extend([-x for x in NASTY_FLOATS])
 NASTY_FLOATS = list(map(float, NASTY_FLOATS))
```

Comment:
Instead of sorting below, `NASTY_FLOATS = sorted(map(float, NASTY_FLOATS), key=flt.float_to_lex)` here.

Reply:
Note that we modify the `NASTY_FLOATS` array after creation to include negative numbers. As it happens that would give the right order with the current sort order of negative numbers, but I think it's clearer to have the sort at that point.

Reply:
The behavior is identical either way, I just prefer `sorted(lst)` to `lst.sort()`.  Not a big deal anyway.